### ISSUE TITLE
BertEmbeddings ConnectionError

### ISSUE THREAD

**Issue by renjunxiang** [zalandoresearch/flair#391]
Hi,
I have downloaded bert-embedding. Because my Internet network is not very stable, using 
``` python
embedding = BertEmbeddings(bert_model='bert-base-uncased')
```
may raise ConnectionError
``` python
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='s3.amazonaws.com', port=443)
```
Is there any way to avoid checking for amazonaws before loading bert-embedding?

**Comment by alanakbik** [comment]
Good point. Do you mean that it does not work unless you are connected to the internet even though you already downloaded the model?

**Comment by renjunxiang** [comment]
Yes, maybe in "pytorch_pretrained_bert\modeling.py", line473,
``` python
        if pretrained_model_name in PRETRAINED_MODEL_ARCHIVE_MAP:
            archive_file = PRETRAINED_MODEL_ARCHIVE_MAP[pretrained_model_name]
        else:
            archive_file = pretrained_model_name
        # redirect to the cache, if necessary
        try:
            resolved_archive_file = cached_path(archive_file, cache_dir=cache_dir)
```
"cached_path" will connect the internet?

And another question:
I found names of bert embeddings are strange, such as "bert-base-uncased" saved as "9c41111e2de84547a463fd39217199738d1e3deb72d4fec4399e6e241983c6f0.ae3cef932725ca7a30cdcb93fc6e09150a55e2a130ec7af63975a16c153ae2ba" and "26bc1ad6c0ac742e9b52263248f6d0f00068293b33709fae12320c0e35ccfbbb.542ce4285a40d23a559526243235df47c5f75c197f04f37d1a0c124c32c9a084".   
It may cause little confusion if I want to use them for something else.

**Comment by alanakbik** [comment]
@renjunxiang have you managed to resolve the problem?

**Comment by renjunxiang** [comment]
Yes, huggingface suggested I could set the pretrained_model_name to a local directory containing a bert_config.json file and a pytorch_model.bin file rather than one of 'bert-base-uncased', 'bert-large-uncased' etc. 
More detailed information is available on <https://github.com/huggingface/pytorch-pretrained-BERT/issues/222>

**Comment by alanakbik** [comment]
Ah great - thanks for sharing! Will close the ticket but feel free to reopen if you have more questions/comments.
